plugins/source/github/gen/recipies/hooks.go

```diff
@@ -37,6 +37,16 @@ func Hooks() []*Resource {
 					Description: "Hook ID",
 					Options:     schema.ColumnCreationOptions{PrimaryKey: true},
 				},
+				codegen.ColumnDefinition{
```

Comment:
cc @candiduslynx with the new tests from the SDK the GitHub plugin tests started failing.
These fields should be resolved to string via the `String()` method